### ISSUE TITLE
Can't find `MonadTrans`

### ISSUE THREAD

**Issue by akrmn** [ndmitchell/hoogle#407]
The following query https://hoogle.haskell.org/?hoogle=MonadTrans%20%2Btransformershows no results, even though `MonadTrans` is definitely in the `transformers` package. Cf https://hoogle.haskell.org/?hoogle=lift%20%2Btransformers which does show the single method of said class. Perhaps Hoogle gets confused by the quantified constraint?

**Comment by Bodigrim** [comment]
Yes, that's a consequence of `HsDecl` from `haskell-src-exts` not able to represent quantified constraints.
